en/flight_modes/orbit.md

```diff
@@ -0,0 +1,46 @@
+# Orbit (MC)
+
+[<img src="../../assets/site/difficulty_easy.png" title="Easy to fly" width="30px" />](../getting_started/flight_modes.md#key_difficulty)&nbsp;[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](../getting_started/flight_modes.md#key_manual)&nbsp;[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
+
+The *Orbit* flight mode allows you to fly a multicopter (or VTOL in multicopter mode) in a circle, yawing so that it always faces towards the centre.
+
+![Orbit Mode - MC](../../assets/flying/orbit.jpg)
+
+*QGroundControl* (or other compatible GCS or MAVLink API) is *required* to enable the mode, and to set the centre position, initial radius and altitude of the orbit.
+Once enabled the vehicle will fly as fast as possible to the closest point on the commanded circle trajectory and do a slow (1m/s) clockwise orbit on the planned circle, facing the center.
+Instructions on how to start an orbit can be found here: [FlyView > Orbit Location](https://docs.qgroundcontrol.com/en/FlyView/FlyView.html#orbit) (*QGroundControl* guide).
+
+> **Note** RC control cannot be used to start the mode (if you switch to the mode via RC it will sit idle).
+
+Once orbit mode is active, the RC controller can be used to change the orbit altitude, radius, speed, and orbit direction:
+- **Left stick:** up/down for altitude change as in [Position mode](../flight_modes/position_mc.md).
+- **Right stick:** right/left controls speed and direction (counter-clockwise/clockwise), up/down controls orbit radius (smaller/bigger).
```

Comment:
Like you wrote in the pr conversation comment https://github.com/PX4/px4_user_guide/pull/472#issuecomment-488187345 the stick ramps up and down the linear velocity (m/s not °/s) and is hence used to accelerate, decelerate, stop, start spinning in the other direction and the speed is kept for further semi-autonomous flight afterwards. Without it keeping the speed by itself it would not be possible to use it "hands-off".

Reply:
@MaEtUgR That isn't quite what I was querying in the above comment, and now I am confused. Can you confirm:

Sticks can either angle control or angle rate control a behaviour - say the speed around the orbit in m/s. 
- If the speed is angle controlled, then if I have the stick at 30% (say) from centre, then the speed will be some fixed value - and as I move it back to the centre the value would reduce. In the centre the speed is zero. If move the stick down then I change the setpoint to a fixed speed in the other direction.
- If the speed is _angle rate_ controlled and I have the stick at 30% say, then this controls the rate of increase of speed. If I leave it there for a bit it will keep increasing. If I move it back to center the speed will stop increasing, but will stay the same. If I move the stick lower then I start applying a negative change to the angle rate.

My believe is that the radius and speed are angle rate controlled - ie if I move the sticks back to the centre or RC drops out the current values will be kept. If I want to change direction, I have to provide a reverse value. That is correct right?

So for right stick I might change
> right/left controls speed and direction (counter-clockwise/clockwise), up/down controls orbit radius (smaller/bigger).

to say:

> right/left controls rate of change of speed in a particular direction (counter-clockwise/clockwise), up/down controls rate of change of orbit radius (smaller/bigger).

Is that correct?

If so, then the diagram is wrong too and I think should be as per the purple writing. Ie the left stick is actually angle controlled
![image](https://user-images.githubusercontent.com/5368500/57207227-453da900-700f-11e9-91d8-96feea858e80.png)

[This implies all my diagrams are wrong for position and altitude modes - these show the altitude as rate controlled when I THINK that actually Yaw is rate controlled on left stick, but altitude is angle controlled - right?]

Reply:
Now I'm also confused.

> If the speed is angle rate controlled then the stick angle controls the rate of change of speed. If I leave it at some positive value for a bit it will keep increasing at the rate defined by the angle. If I move it back to center the speed will stop increasing, but will stay the same final speed. If I move the stick lower then I start applying a negative change to the angle rate and will slow it down.

That's correct.

Let me write it as clear as possible:

|                        | Centered/RC loss                    | Deflected Down/Left                     | Deflected Up/Right                   |
|------------------------|-------------------------------------|-----------------------------------------|--------------------------------------|
| Left stick down/up     | Altitude is locked                  | Vehicle has downwards speed             | Vehicle has upwards speed            |
| Left stick left/right  | -                                   | -                                       | -                                    |
| Right stick down/up    | Keep current radius                 | Radius gets bigger until maximum (100m) | Radius gets lower until minimum (1m) |
| Right stick left/right | Keep current m/s speed around orbit | Accelerate CW/Decelerate CCW *          | Decelerate CW/Accelerate CCW *       |

* When Decelerating to zero speed and staying on the stick it will switch direction of rotation and accelerate again. Maximum velocity is 10m/s and it's even more limited to keep the centripetal acceleration below 2m/s^2.

Reply:
@MaEtUgR Ha ha. I like the table - will steal it.
Still need to know whether rate or angle controlled because that affects the diagrams. Fortunately this is now mostly answered - see queries inline.

From the table I can say that 
- altitude (Left stick Down/UP) is angle controlled. The speed setpoint is mapped to the angle of the stick.
- radius (Right stick down up) looks like angle rate control -ish
  - does the rate of radius change change based on the stick angle or is it basically a fixed rate of change when stick is not centred?
 - speed of rotation (Right stick, left right) - this looks angle rate controlled, with acceleration based on stick position, but limited as you indicated?

Upshot, I do need to change the left stick labels. 
For right stick, depends on your answer above, but probably left/right at least is rate controlled.